apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx

```diff
@@ -0,0 +1,55 @@
+import { useContext, useState } from 'react';
+import { dateConvertor, getName } from '../utils';
+import { useQuery } from 'react-query';
+import { AppContext, IAppContext } from '../../../context';
+
+export function useFetchUsers() {
+    const { api } = useContext<IAppContext>(AppContext);
+
+    const [paginationCnfg, setPaginationCnfg] = useState({ page: 0, size: 5 });
+
+    const { data: users, isLoading: isUsersLoading }: any = useQuery(
+        ['users', paginationCnfg],
+        () => api?.users.readList(paginationCnfg),
+        {
+            keepPreviousData: true,
+            select: (users: any) => {
+                return users.data.map((user: any) => {
+                    return {
+                        ...user,
+                        username: getName(user.userName, user.firstName, user.secondName),
```

Comment:
If you want to use this utils method pass only one argument user: getUserName(user)

Reply:
done